roles/lib_openshift/src/ansible/oc_adm_registry.py

```diff
@@ -24,8 +24,8 @@ def main():
             service_account=dict(default='registry', type='str'),
             mount_host=dict(default=None, type='str'),
             volume_mounts=dict(default=None, type='list'),
-            env_vars=dict(default=None, type='dict'),
-            edits=dict(default=None, type='list'),
+            env_vars=dict(default={}, type='dict'),
+            edits=dict(default=[], type='list'),
```

Comment:
@detiber I think it's OK because it should be redefined every time ``main`` is called. If it was at a higher scope I think we'd hit that problem.